src/GitVersion.Core/PublicAPI.Unshipped.txt

```diff
@@ -0,0 +1 @@
+GitVersion.RepositoryStore.ExcludingBranches(System.Collections.Generic.IEnumerable<GitVersion.IBranch!>! branchesToExclude, bool excludeRemotes = false) -> System.Collections.Generic.IEnumerable<GitVersion.IBranch!>!
```

Comment:
if we approve this, I suppose I have to move this inside `PublicAPI.Shipped.txt` isn't it?